### PR TITLE
A lot of small fixes

### PR DESCRIPTION
**Pull request by Drapegnik**
https://github.com/babajka/babajka-backend/pull/57

## fix meta keywords
Screen Shot 2019-09-11 at 00 27 58

### before
Screen Shot 2019-09-12 at 18 57 52

### after
Screen Shot 2019-09-12 at 18 59 33

### localize motto, add tag links
Screen Shot 2019-09-12 at 18 54 46

## fix bug with tag page blocks rendering
Screen Shot 2019-09-12 at 20 00 04

## fix input spinner
[Image from Gyazo](https://gyazo.com/9ed8d783778ce6ac33ca4bbf171c5d93)

## chore

- add `useLocalization` & `useLocaleContext`
- remove some legacy code
- replace `topic` with `topicSlug` for `tags`
- add `tagsByTopic`
- add [sidebar close](https://github.com/babajka/babajka-frontend/pull/66#issuecomment-529147051)

### CODE REVIEW

**Comment by Drapegnik** [comment]
[to make wir.by searchable by keyword "вір"](https://trello.com/c/f07IFiUR/296-to-make-wirby-searchable-by-keyword-%D0%B2%D1%96%D1%80)

**Comment by Drapegnik** [comment]
[i18n hook](https://trello.com/c/hddC6BfO/378-i18n-hook)

**Comment by Drapegnik** [comment]
[ligthhouse audit fixes](https://trello.com/c/6vOZ3Jqd/397-ligthhouse-audit-fixes)

**Comment by Drapegnik** [comment]
[fix input spinner](https://trello.com/c/Es4QiXKz/363-fix-input-spinner)

File: constants/index.js [hunk 1]
```diff
@@ -28,8 +28,6 @@ const STATIC_PATHS = ['static', '_next'];
 
 const TOPICS = ['themes', 'locations', 'times', 'personalities', 'authors', 'brands'];
 
-const FIBERY_HOST = 'https://wir.fibery.io';
```
Comment (UladBohdan) [inline constants/index.js:31]: so with this const being removed, Frontend does not communicate with Fibery directly anymore? that's cool

File: constants/misc.js [hunk 2]
```diff
@@ -4,6 +4,4 @@ import { TOPICS } from 'constants';
 
 export const TOPIC = createConstants(...TOPICS);
 
-export const TAG_BLOCK_SIZE = 10;
-
-export const TAG_LEVELS_SIZES = [2, 3, 2, 3];
+export const a = 1;
```
Comment (UladBohdan) [inline constants/misc.js:7]: what's that? :)

File: pages/_document.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <body>
+          <Main />
+          <NextScript />
+        </body>
+      </html>
+    );
+  }
+}
+
+export default MyDocument;
```
Comment (UladBohdan) [inline pages/_document.js:27]: I am not sure what is this file for.
Are you ensuring the structure of the document created by Next.js ?
Why were we ok without it before?

File: constants/misc.js (hunk 2 above)
Comment (UladBohdan) [reply constants/misc.js:7]: https://a1.by easter egg

**Review by UladBohdan — APPROVED** [review]

File: pages/_document.js (hunk 3 above)
Comment (Drapegnik) [reply pages/_document.js:27]: @UladBohdan, https://github.com/zeit/next.js#custom-document

I added it to pass `lang` prop to `<html />`

File: constants/misc.js (hunk 2 above)
Comment (Drapegnik) [reply constants/misc.js:7]: bypass `eslint` rule about default export

**Pull request merged**